Primo.js

```diff
@@ -351,11 +418,18 @@ function importPNX(text) {
 }
 
 function stripAuthor(str) {
+	// Example "Andrea : da Barberino (1370-circa 1431)"
+	// http://gutenberg.beic.it/webclient/MetadataManager?pid=3836862&descriptive_only=true
 	return str
-		// Remove year
-		.replace(/\s*,?\s*\(?\d{4}-?(\d{4})?\)?/g, '')
 		// Remove things like (illustrator). TODO: use this to assign creator type?
-		.replace(/\s*,?\s*\([^()]*\)$/, '');
+		// Also matches an empty parenthesis
+		.replace(/\s*,?\s*\([^()]*\)$/, '')
+		// Remove almost any remaining parenthesis, starting with numbers (dates) and
+		// letters used for common international abbreviations about dates
+		.replace(/\([\d\sabcefilr.? ]*[-–][\d\abcefilr.? ]*\)/gi, '')
```

Comment:
Was the comment https://github.com/zotero/translators/pull/822#discussion_r22402268 the last one on this issue?